src/main/java/racingcar/domain/Car.java

```diff
@@ -1,69 +1,43 @@
 package racingcar.domain;
 
 import java.util.Objects;
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
 import racingcar.dto.CarStatus;
 
 public class Car implements Comparable<Car> {
-    private static final Pattern NAME_PATTERN = Pattern.compile("[^ㄱ-ㅎ가-힣a-zA-Z0-9_-]");
-    private static final int MAX_NAME_LENGTH = 5;
     private static final int CAR_MOVE_THRESHOLD = 4;
 
-    private final String name;
-    private int position = 0;
+    private final CarName name;
+    private Position position;
 
-    public Car(final String name) {
-        validateName(name);
-        this.name = name.trim();
+    private Car(final CarName name) {
+        this.name = name;
+        this.position = new Position(0);
```

Comment:
0을 가진 Position 객체를 여러번 사용될 것으로 예상돼요!
`Position.ZERO` 이런식으로 객체를 재활용 해보면 어떨까요!

```suggestion
        this.position = Position.ZERO;
```

Reply:
좋은 의견 감사합니다!
바로 반영해 보았는데 의도대로 잘 반영된 것인지 확인하여 주시면 감사하겠습니다. 😊

Reply:
리브가 만족하면 된다고 생각합니다 😄 
(제가 생각했던 방향은 맞아요 ㅎㅎ!)